radsat-sk/operation/subsystems/RAntenna.c

```diff
@@ -2,6 +2,8 @@
  * @file RAntenna.c
  * @date Jan 15, 2022
  * @author Addi Amaya (caa746)
+ * @edit date - May 7, 2022
+ * @edit author - Atharva (iya789)
```

Comment:
I would remove the edit date; probably not necessary. And instead of adding your name on its own line, just add your name next to Addi's e.g.:
```c
 * @auther Addi Amaya (caa746) and Atharva Kulkarni (ayi789)
```
I would request you put your full name as well, unless you're not comfortable doing so